AdminUi/apps/admin_ui/lib/core/modals/settings_dialog.dart

```diff
@@ -18,24 +18,29 @@ class _SettingsDialog extends StatelessWidget with WatchItMixin {
     final themeMode = watchValue((ThemeModeModel x) => x.themeMode);
 
     return AlertDialog(
-      title: Text(context.l10n.settings),
-      content: Column(
-        mainAxisSize: MainAxisSize.min,
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: [
-          Text(context.l10n.theme, style: Theme.of(context).textTheme.bodyLarge),
-          Gaps.h4,
-          SegmentedButton(
-            showSelectedIcon: false,
-            segments: [
-              ButtonSegment(value: ThemeMode.light, icon: const Icon(Icons.light_mode), label: Text(context.l10n.light)),
-              ButtonSegment(value: ThemeMode.system, icon: const Icon(Icons.settings), label: Text(context.l10n.system)),
-              ButtonSegment(value: ThemeMode.dark, icon: const Icon(Icons.dark_mode), label: Text(context.l10n.dark)),
-            ],
-            selected: {themeMode},
-            onSelectionChanged: (selected) => GetIt.I<ThemeModeModel>().setThemeMode(selected.first),
-          ),
-        ],
+      shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(8)),
+      title: Text(context.l10n.settings, textAlign: TextAlign.center),
+      content: Padding(
+        padding: const EdgeInsets.only(bottom: 32),
```

Comment:
you should use `AlertDialog#contentPadding` instead to remove on layer of widgets, same for all other dialogs, make sure to use `const EdgeInsets.only(left: 24, right: 24, top: 20, bottom: 32)` to not override all defaults and only change the bottom.

See https://api.flutter.dev/flutter/material/AlertDialog/contentPadding.html for full docs

Reply:
again, ping me if done

Reply:
@jkoenig134